### ISSUE TITLE
Motion to Vacate | Update Return to Motions Attorney

### ISSUE THREAD

**Issue by leikkisa** [department-of-veterans-affairs/caseflow#13862]
## User or job story
User story: As a judge, I would like to be able to return a motion to vacate back to the motions attorney for reasons other than not having the draft recommendation document.

## Acceptance criteria
- [ ] Please put this work behind the feature toggle: [review_motion_to_vacate]
- [ ] Judge has the option to return to the motions attorney for all vacate types (always show the yellow alert component)
- [ ] Update copy on the yellow alert
- [ ] Update copy on the new task modal
- [ ] Include screenshot(s) in the Github issue
- [ ] Update documentation: [Motions to Vacate](https://github.com/department-of-veterans-affairs/caseflow/wiki/Motions-to-Vacate)

### Designs
Updated Copy
**Yellow alert:**
If you weren't provided the draft ruling letter, or would like to request that the motions attorney make changes, please return to the motions attorney.

**Task modal:**
Use this action to return the Motion to Vacate task to the previous Motions Attorney in order to request changes to the ruling letter draft, or if the ruling letter draft is missing.

#### Current designs
![Screen Shot 2020-04-01 at 2.51.01 PM.png](https://images.zenhubusercontent.com/58f6b6eb51aa2c30577bb60e/8e3e32b1-9d1b-4b5e-ba4f-a3ae6dcd2518)

![Screen Shot 2020-04-01 at 2.51.36 PM.png](https://images.zenhubusercontent.com/58f6b6eb51aa2c30577bb60e/30dd0f12-3e62-4f8f-b81b-bf3c76b089e9)

### Technical notes
This is related to ticket: #13861 but they can be worked on in parallel.

No changes to the functionality are needed for this ticket.

### Background/context
This ticket is being created as a result of a demo to stakeholders on March 30, 2020. [See Google Doc with findings](https://docs.google.com/document/d/1w-3MhdJMtGwz9DBEhQC8vgbbgbNHA-Bz_FWqVpMV1pQ/edit#).

**Comment by leikkisa** [comment]
@rutvigupta-design Tagging you for review on this ticket, in case you have a different understanding or have any edits to the copy suggestions.  Thanks!

**Comment by rutvigupta-design** [comment]
@leikkisa as per discussion with @msj0nes I will explore a design variation for this and get feedback from a few stakeholders!

**Comment by rutvigupta-design** [comment]
**Note**: This is currently blocked until we get feedback from stakeholders next week. cc @msj0nes

**Comment by msj0nes** [comment]
@leikkisa I would move forward with the updated copy mentioned above and unblock. Any feedback received after can be explored.

**Comment by jcq** [comment]
@msj0nes @leikkisa 
Just to clarify, this means that we're also removing this bit from the modal copy?
```If the previous attorney is inactive, this will return to the Litigation Support team queue for reassignment.```

**Comment by jcq** [comment]
@msj0nes 

And one more question...

Since the copy for the alert no longer refers to a given disposition, do we want this to be visible even if the judge hasn't yet chosen a disposition? Or keep the current behavior of only showing once the judge has selected a disposition? (I'm assuming the former makes more sense, given the other changes...?)

**Comment by msj0nes** [comment]
@jcq this copy should remain in the modal

```If the previous attorney is inactive, this will return to the Litigation Support team queue for reassignment.```

**Comment by msj0nes** [comment]
@jcq confirming we are referring to the same thing below. Lets reduce complexity behind it.

**Comment by jcq** [comment]
## Updated Screenshots

#### Alert
<img width="548" alt="Screen Shot 2020-04-13 at 14 19 26" src="https://user-images.githubusercontent.com/2581274/79170586-b13e9a80-7da4-11ea-9911-8c6c08567dff.png">

---

#### Modal
<img width="521" alt="Screen Shot 2020-04-13 at 14 20 21" src="https://user-images.githubusercontent.com/2581274/79170608-be5b8980-7da4-11ea-8b14-4a274cd00fb5.png">
